openedx/core/djangoapps/content_libraries/permissions.py

```diff
@@ -41,6 +42,12 @@
 )
 
 
+# Are we in Studio? (Is there a better or more contextual way to define this, e.g. get from learning context?)
```

Comment:
You could also check for `settings.SERVICE_VARIANT` for 'cms' vs. 'lms'.

Reply:
Sure, that seems better to me even though we don't seem to be using that pattern much in the python code.